modules/tinymce/src/themes/silver/main/ts/ui/dialog/IFrame.ts

```diff
@@ -8,62 +8,44 @@
 import { Dialog } from '@ephox/bridge';
 import { Cell, Optional } from '@ephox/katamari';
-import { PlatformDetection } from '@ephox/sand';
 import { Attribute } from '@ephox/sugar';
 
 import { UiFactoryBackstageProviders } from '../../backstage/Backstage';
 import { renderFormFieldWith, renderLabel } from '../alien/FieldLabeller';
 import { RepresentingConfigs } from '../alien/RepresentingConfigs';
 import * as NavigableObject from '../general/NavigableObject';
 
-const platformNeedsSandboxing = !(PlatformDetection.detect().browser.isIE() || PlatformDetection.detect().browser.isEdge());
-
 interface IFrameSourcing {
   getValue: (frame: AlloyComponent) => string;
   setValue: (frame: AlloyComponent, value: string) => void;
 }
 
 type IframeSpec = Omit<Dialog.Iframe, 'type'>;
 
-const getDynamicSource = (isSandbox): IFrameSourcing => {
+const getDynamicSource = (): IFrameSourcing => {
   const cachedValue = Cell('');
   return {
     getValue: (_frameComponent: AlloyComponent): string =>
-      // Ideally we should fetch data from the iframe...innerHtml, this triggers Corrs errors
+      // Ideally we should fetch data from the iframe...innerHtml, this triggers Cors errors
       cachedValue.get(),
     setValue: (frameComponent: AlloyComponent, html: string) => {
-
-      if (!isSandbox) {
-        Attribute.set(frameComponent.element, 'src', `javascript:''`);
-        // IE 6-11 doesn't support data uris on iframeComponents
-        // and Edge only supports upto ~4000 chars in data uris
-        // so I guess they will have to be less secure since we can't sandbox on those
-        // TODO: Use sandbox if future versions of IE/Edge supports iframeComponents with data: uris.
-        const doc = frameComponent.element.dom.contentWindow.document;
-
-        doc.open();
-        doc.write(html);
-        doc.close();
-
-      } else {
-        // TINY-3769: We need to use srcdoc here, instead of src with a data URI, otherwise browsers won't retain the Origin.
-        // See https://bugs.chromium.org/p/chromium/issues/detail?id=58999#c11
-        Attribute.set(frameComponent.element, 'srcdoc', html);
-      }
+      // TINY-3769: We need to use srcdoc here, instead of src with a data URI, otherwise browsers won't retain the Origin.
+      // See https://bugs.chromium.org/p/chromium/issues/detail?id=58999#c11
+      Attribute.set(frameComponent.element, 'srcdoc', html);
       cachedValue.set(html);
     }
   };
 };
 
 const renderIFrame = (spec: IframeSpec, providersBackstage: UiFactoryBackstageProviders) => {
-  const isSandbox = platformNeedsSandboxing && spec.sandboxed;
+  const isSandbox = spec.sandboxed;
 
   const attributes = {
     ...spec.label.map<{ title?: string }>((title) => ({ title })).getOr({}),
     ...isSandbox ? { sandbox: 'allow-scripts allow-same-origin' } : { }
   };
 
-  const sourcing = getDynamicSource(isSandbox);
+  const sourcing = getDynamicSource();
```

Comment:
Don't we still need to pass the sandbox in through here, just in case the spec asked for sandboxing?

Reply:
No, if you look at the reason it was done it was for IE 11/Legacy Edge. It looks like someone originally used that because sandboxing was always disabled for those 2 browsers.

Reply:
I should also add we're fine using `srcdoc` regardless of if we're using a sandbox or not. It'll also avoid those verbose console warnings modern browsers show about `document.write` being deprecated. e.g:
```
[Violation] Avoid using document.write(). https://developers.google.com/web/updates/2016/08/removing-document-write
```

Reply:
I guess we should change the init of tinymce to not use document.write as well and use srcdoc.

Reply:
Yeah that's a good point... I actually did this like a year ago so I might have an old branch lying around somewhere 🤔 either way I'll log a Jira

Reply:
I ended up updating the init sequence to use `srcdoc` as part of this instead since it's also related to IE 11 cleanup/modernisation.